### PR TITLE
Googlenet

### PR DESCRIPTION
**Pull request by albertkao227**
My first draft.

### CODE REVIEW

File: chapters/en/Unit 2 - Convolutional Neural Networks/googlenet.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# GoogLeNet
+
+### 1. Introduction 
+
+Inception is a convolution neural network (CNN) architecture, used for computer vision classification and detection tasks. It has less than 7 million parameters, which is 9 times smaller than AlexNet, and 22 times smaller than VGG16. In ImageNet 2014 challenge, the authors from Google submitted a version of Inception network callled GoogLeNet (an homage to LeNet), which achieved state-of-the-art results with less parameters than prior best methods.   
+
+### 2. Key Architecture 
+
+Before Inception, successful network architectures such as AlexNet or VGG have shown the importance of increasing network depth. However, increasing network depth also increases computation steps, and problems such as overfitting and vanishing gradient may occur. Inception network introduced a method to efficiently train complex CNNs wih less floating parameters.
```
Comment (sezan92) [inline chapters/en/Unit 2 - Convolutional Neural Networks/googlenet.mdx]:
```suggestion
Before the Inception model, the successful  architectures such as AlexNet or VGG have shown the importance of increasing the network depth. However, increasing the  depth also increases computational complexity, and problems such as overfitting and/or vanishing gradient may occur. The Inception model introduced a method to efficiently train complex CNNs wih less floating parameters.   
```

**Review by sezan92 — CHANGES_REQUESTED** [review]
Looks good. Except I feel there are some grammatical mistakes. Is it possible to update using Grammarly free version?

**Comment by albertkao227** [comment]
Thanks @sezan92 , revised the sections.

File: chapters/en/Unit 2 - Convolutional Neural Networks/googlenet.mdx (hunk 1 above)
Comment (albertkao227) [reply chapters/en/Unit 2 - Convolutional Neural Networks/googlenet.mdx]: revised this section

File: chapters/en/Unit 2 - Convolutional Neural Networks/googlenet.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# GoogLeNet
+
+### 1. Overview
+
+The Inception architecture, a type of convolutional neural network (CNN) designed for tasks in computer vision such as classification and detection, stands out due to its efficiency. Containing fewer than 7 million parameters, it is significantly more compact than its predecessors, being 9 times smaller than AlexNet and 22 times smaller than VGG16. This architecture gained recognition in the ImageNet 2014 challenge, where Google's adaptation, named GoogLeNet (a tribute to LeNet), set new benchmarks in performance while utilizing fewer parameters compared to previous leading methods.
+
+### 2. Architectural Innovations
+
+Prior to the advent of the Inception architecture, models like AlexNet and VGG demonstrated the benefits of deeper network structures. However, deeper networks typically entail more computational steps and can lead to issues such as overfitting and the vanishing gradient problem. The Inception architecture offers a solution, enabling the training of complex CNNs with a reduced count of floating-point parameters.
```
Comment (sitammeur) [inline chapters/en/Unit 2 - Convolutional Neural Networks/googlenet.mdx]:
```suggestion
Before the advent of the Inception architecture, models like AlexNet and VGG demonstrated the benefits of deeper network structures. However, deeper networks typically entail more computational steps and can lead to issues such as overfitting and the vanishing gradient problem. The Inception architecture offers a solution, enabling the training of complex CNNs with a reduced count of floating-point parameters.
```

**Review by sitammeur — COMMENTED** [review]
The writing is good overall!

**Review by JvThunder — APPROVED** [review]
Very good and concise summary of the paper. I have a question though, you wrote of using 9 inception layers yet you only put 8 in the code. You might have missed one layer.

**Review by sezan92 — APPROVED** [review]

**Pull request merged**